moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -568,13 +568,18 @@ void JogCalcs::enforceJointVelocityLimits(Eigen::VectorXd& calculated_joint_chan
 bool JogCalcs::checkIfJointsWithinURDFBounds(trajectory_msgs::JointTrajectory& new_jt_traj)
 {
   bool halting = false;
+
+  if (new_jt_traj.points[0].velocities.size() < num_joints_)
+    new_jt_traj.points[0].velocities.resize(num_joints_);
```

Comment:
Would prefer to delete this and move the check to line 584. I think that's a cleaner way

Reply:
agreed. I just made local changes to do exactly that.